src/wllama.ts

```diff
@@ -206,6 +206,33 @@ export class Wllama {
     return this.useMultiThread;
   }
 
+  /**
+   * Parses a model URL and returns an array of URLs based on the following patterns:
+   * If the input URL is an array, it returns the array itself.
+   * If the input URL is a string in the `gguf-split` format, it returns an array containing the URL of each shard in ascending order.
+   * Otherwise, it returns an array containing the input URL as a single element array.
+   */
+  parseModelUrl(modelUrl: string | string[]) {
```

Comment:
```suggestion
  private parseModelUrl(modelUrl: string | string[]): string[] {
```

Also maybe add `@param` for the inline docs